src/Http/Controllers/Controller.php

```diff
@@ -58,4 +67,32 @@ protected function getUserFromRoute(): Model
 
         return $modelClass::findOrFail($userId);
     }
+
+    /**
+     * Tries to retrieve the model.
+     *
+     * @param  mixed  $value
+     * @param  string|null  $modelClass
+     * @return \Illuminate\Database\Eloquent\Model
+     *
+     * @throws \Illuminate\Database\Eloquent\ModelNotFoundException
+     */
+    public function getModel($value, string $modelClass = null): Model
```

Comment:
Code got improved in [this PR](https://github.com/owowagency/laravel-gossip/pull/13/files#diff-06fe6c5c07e5f6748059572da347bdf0fd666746b5d98818eed6fd7bebf3f8cbR12).